doc/world_format.md

```diff
@@ -298,41 +298,56 @@ Before 5.12.0 it looked like this:
 CREATE TABLE `blocks` (`pos` INT NOT NULL PRIMARY KEY, `data` BLOB);
 ```
 
+Improved performance in SQLite3 for existing worlds can be obtained by
```

Comment:
off-topic in this documentation, IMO

Reply:
Sure but they wouldn't read `world_format.md`.
Also this advice isn't even applicable to any new worlds after 5.12.0 so I don't really think it's worth documenting.